src/runners/test_alert_queries_runner.py

```diff
@@ -0,0 +1,61 @@
+from runners import alert_queries_runner
+from runners import test_queries
+from runners.helpers import db
+import os
+import json
+import pytest
+
+TEST_1_OUTPUT = {"ACTION": "test action 1",
+                 "ACTOR": "test_actor",
+                 "DESCRIPTION": "This is a test alert query; this should be grouped with Test 3",
+                 "DETECTOR": "SnowAlert",
+                 "ENVIRONMENT": {"account": "account_test", "cloud": "cloud_test"},
+                 "EVENT_DATA": {"data": "test data"},
+                 "OBJECT": ["obj1", "obj2"],
+                 "QUERY_ID": "test_1_query",
+                 "QUERY_NAME": "TEST1_ALERT_QUERY",
+                 "SEVERITY": "low",
+                 "SOURCES": ["source"],
+                 "TITLE": "test1_alert_query"}
+
+
+def preprocess():
+    ctx = db.connect()
+    assert ctx is not None
+    db.execute(ctx, 'truncate table results.alerts')
```

Comment:
this should happen in a tear-down that is guaranteed to be run after a test completes, not "proactively" after other tests leave behind a mess. in general, it's the responsibility of a test suite / test to make sure that it leaves the testing area clean, not the next tests.

this way, the test encapsulates both what it's testing and its cleanup. in this test suite, for example, we should make sure we are cleaning up metadata rows.